frontend/src/pages/settings/users/index.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <SettingsPage page-title="settings.users.users">
+    <template #actions>
+      <VBtn
+        color="primary"
+        variant="elevated"
+        class="ml-a"
+        @click="$router.push('/settings/users/new')">
+        {{ t('settings.users.new') }}
+      </VBtn>
+      <VBtn
+        variant="elevated"
+        href="https://jellyfin.org/docs/general/server/users/adding-managing-users"
+        rel="noreferrer noopener"
+        target="_blank">
+        {{ t('settings.help') }}
+      </VBtn>
+    </template>
+    <template #content>
+      <VCol>
+        <VTable>
+          <thead>
+            <tr>
+              <th
+                v-for="{ text, value } in headers"
+                :id="value"
+                :key="value">
+                {{ text }}
+              </th>
+              <th scope="col">
+              <!--for avatar-->
+              </th>
+              <th scope="col">
+              <!--for edit btn-->
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr
+              v-for="user in users"
+              :key="user.Id ?? undefined">
+              <td
+                v-for="{ value } in headers"
+                :key="value">
+                {{
+                  value !== 'LastActivityDate'
+                    ? user[value]
+                    : useDateFns(
+                      formatRelative,
+                      parseJSON(user[value] ?? "2023-02-02T02:02:02Z"), //FIXME: using 'unknown' doesnt work here? its used somewhere else in the code as well, does that even work there?
```

Comment:
Not sure what's going on here, but when tried using `"unknown"` instead of the hard-coded date I got this error: 
```
Uncaught (in promise) RangeError: Invalid time value
```
This happens for users who haven't logged in yet. I copied this code from somewhere else in this codebase, so if this gets fixed here that fix should also be tested there.

Reply:
How are you able to access this page if you're not logged in yet?

Reply:
I don't understand your question. This is an admin page and should only be accessible by logged-in administrators, right? I didn't implement permission checks since those are on the server side. Do you want me to implement some code that redirects unauthorized/unauthenticated users back to the index/login page?

Reply:
You forgot to add [this](https://github.com/jellyfin/jellyfin-vue/blob/master/frontend/src/pages/settings/apikeys.vue#L101C1-L104C9)

As you said above, permissions are enforced server-side, but what's the point of letting normal users access stuff that they can't change?

Reply:
Wait, but I do have that in line 76, no?

Reply:
@rdmchr My bad for everything: I thought you meant that you were able to access the route when no user was logged into the client (by inputting the exact url). Now I understand you mean **the users that are being managed**.

Any reason I'm missing why you don't do this like [web](https://github.com/jellyfin/jellyfin-web/blob/master/src/components/dashboard/users/UserCardBox.tsx#L26)? Fired up a quick Codespace to check and it works fine with the same functions used and I'm not seeing any shortcomings with it.

Reply:
No, I made a mistake here. I copied that code from https://github.com/rdmchr/jellyfin-vue/blob/9a1f88c5d61b0f3f53268b0a604f5be7bfff8f6c/frontend/src/pages/settings/apikeys.vue#L44-L50 and in that case, the `'unknown'` will "never" be the outcome, since every key has a date created, but not every user has a date when they were last seen (ex. newly created keys have a created date, freshly created users were never seen). 
So in the original place that code should work.

I think we should create some utility function for this use case which handles it correctly for both cases.

Reply:
@rdmchr And do you see any shortcoming to the Web's implementation? I think it's better to use browser's native functions when possible, they should be less prone to errors.

Reply:
@ferferga I implemented a basic method. Currently, you have to use it like this:
```Vue
          <VCardSubtitle>
            {{ $t('settings.users.lastActivityDate', {
              value: user.LastActivityDate ? getRelativeTime(new Date(user.LastActivityDate)) : t('unknown')})
            }}
          </VCardSubtitle>
````
To get this result:
![image](https://github.com/jellyfin/jellyfin-vue/assets/66096031/d4341205-a10b-44b2-b991-071a3eee808b)

Might be worth adding all that additional stuff to the method as well since I don't see any use case where you would want just the relative time.

Reply:
Two things about the text:
1. "Last seen unknown ago" sounds somewhat good in English, but with it we're overriding the date-fns translations and adding extra complexity. In Spanish, your sentence would be "Última vez hace desconocido" which is completely wrong. This solely makes me reluctant to add this in this state.
2. "unknown" should be lowercased.
3. You're assuming we don't know the date of the last login, while in fact the missing property means "this user has no logged-in yet". I don't know if there are other cases where the server can return ``undefined | null`` for this, but I can't imagine another. When a user hasn't logged-in, that's **a very precise information you know**, not *unknown*.

I'm divided about this, I think the safest approach would be to do it like web (skip displaying the text altogether) but the most concise one would be to display the "not logged-in text", even though it might have some edge cases we're not aware of.

Also, I'm not a fan of having an exclusive "shortcut" function for something that's not repeated many times in the codebase. Though I was unsure of what you mean, I thought you would add additional processing to the function, but in this case I think you're better off leaving it as is (or extracting the function from the template and putting it inside ``script setup``).

Reply:
Yeah, you're right, I have removed the function and updated the page according to your recommendations.